src/test/java/bio/terra/service/dataset/flight/inheritsteward/InheritStewardSetFlagStepTest.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.service.dataset.flight.inheritsteward;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import bio.terra.common.category.Unit;
+import bio.terra.service.dataset.DatasetDao;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@Tag(Unit.TAG)
+class InheritStewardSetFlagStepTest {
+
+  @Mock private DatasetDao datasetDao;
+  private UUID datasetId;
+  private boolean enableInheritSteward;
+  private InheritStewardSetFlagStep step;
+  private FlightContext context;
+
+  @BeforeEach
+  void setUp() {
+    datasetId = UUID.randomUUID();
+    enableInheritSteward = true;
+    step = new InheritStewardSetFlagStep(datasetDao, datasetId, enableInheritSteward);
+    context = mock(FlightContext.class);
+  }
+
+  @Test
+  void testDoStepSuccess() {
```

Comment:
The TDR code base is inconsistent with regard to test method names, but the practice of prepending `test` comes from the days before JUnit supported annotations; back then a test method had to start with `test` for JUnit to run it. Since the method is already in a class with `Test` in its name, I don't think a test method should start or end with `Test`.

My preferred convention is to name the test with the same name as the method being tested. The main positive test would be the method name, and various negative tests (or additional positive tests) would ave more suffixes added to describe what's different. Some people use the convention of adding an underscore to separate the names, but since that's not a valid name in our code style I don't think that's a good approach.
```suggestion
  void doStep() {
```